src/controllers/playback/video/index.js

```diff
@@ -209,8 +209,13 @@ import { setBackdropTransparency, TRANSPARENCY_LEVEL } from '../../../components
             if (!itemName) {
                 itemName = parentName || '';
             }
-
-            LibraryMenu.setTitle(itemName);
+            //Display the item with its premiere date if it has one
+            if ('PremiereDate' in item) {
+                const date = datetime.parseISO8601Date(item.PremiereDate).getFullYear();
```

Comment:
`parseISO8601Date` is mostly used in the `try/catch` block because it throws an exception.

How about this?
```js
let title = itemName;

if ('PremiereDate' in item) {
    try {
        const year = datetime.parseISO8601Date(item.PremiereDate).getFullYear();
        title += ` ${year}`;
    } catch () {
        // print error
    }
}

LibraryMenu.setTitle(title);
```

Reply:
I see. I'll update the code now.